python/lsst/qserv/tests/benchmark.py

```diff
@@ -145,6 +145,9 @@ def runQueries(self, stopAt=MAX_QUERY):
                                       str(pragmas))
                     column_names = 'noheader' not in pragmas
                     sqlInterface.execute(qText, outFile, column_names)
+                    if 'sortresult' in pragmas:
+                        sortcmd = 'sort %s -o %s' % (outFile, outFile)
+                        os.system(sortcmd)
```

Comment:
Well, it's one line to check the return value so you can know exactly why it's failing before downstream tests fail. I still don't like using a shell function for this.